examples/leg_3DoF_example.cpp

```diff
@@ -46,7 +46,7 @@ class Joints3DoF : public kodlab::mjbots::MjbotsControlLoop<ManyMotorLog> {
     for (int servo = 0; servo < num_joints_; servo++) {
       log_data_->positions[servo]  = robot_->GetJointPositions()[servo];
       log_data_->velocities[servo] = robot_->GetJointVelocities()[servo];
-      log_data_->modes[servo] = static_cast<int>(mjbots_interface_->GetJointModes()[servo]);
+      log_data_->modes[servo] = static_cast<int>(robot_interface_->GetJointModes()[servo]);
```

Comment:
this a problem for later, but "mode" is a mjbots specific state
and maybe should be thought of as such, not sure what that means for logging. For now, simulation should just send some false modes (maybe on and stop but no errors)

Reply:
Super low priority